src/main/webapp/style.css

```diff
@@ -1,9 +1,9 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+html {
+    box-sizing: border-box;
 }
```

Comment:
What is this CSS for? What does it do? I trust you but I just don't know.

Reply:
This makes it such that the padding and border are part of the width and height for the html document.